### DISCUSSION TITLE
Welcome to Quantum-Wormhole Discussions!

### DISCUSSION THREAD

**Discussion by AdhesionTek** [AdhesionTek/Quantum-Wormhole#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kelejun** [comment]
我超，还真有

草，还用的是 WTFPL 协议🤣

**Comment by Nigel-WANGJun** [comment]
:cry:

**Comment by Wenbobobo** [comment]
⣿⣆⠱⣝⡵⣝⢅⠙⣿⢕⢕⢕⢕⢝⣥⢒⠅⣿⣿⣿⡿⣳⣌⠪⡪⣡⢑
⣿⣿⣦⠹⣳⣳⣕⢅⠈⢗⢕⢕⢕⢕⢕⢈⢆⠟⠋⠉⠁⠉⠉⠁⠈⠼⢐
⢰⣶⣶⣦⣝⢝⢕⢕⠅⡆⢕⢕⢕⢕⢕⣴⠏⣠⡶⠛⡉⡉⡛⢶⣦⡀⠐
⡄⢻⢟⣿⣿⣷⣕⣕⣅⣿⣔⣕⣵⣵⣿⣿⢠⣿⢠⣮⡈⣌⠨⠅⠹⣷⡀
⡵⠟⠈⢀⣀⣀⡀⠉⢿⣿⣿⣿⣿⣿⣿⣿⣼⣿⢈⡋⠴⢿⡟⣡⡇⣿⡇
⠁⣠⣾⠟⡉⡉⡉⠻⣦⣻⣿⣿⣿⣿⣿⣿⣿⣿⣧⠸⣿⣦⣥⣿⡇⡿⣰
⢰⣿⡏⣴⣌⠈⣌⠡⠈⢻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣬⣉⣉⣁⣄⢖⢕
⢻⣿⡇⢙⠁⠴⢿⡟⣡⡆⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣷⣵
⣄⣻⣿⣌⠘⢿⣷⣥⣿⠇⣿⣿⣿⣿⣿⣿⠛⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿
⢄⠻⣿⣟⠿⠦⠍⠉⣡⣾⣿⣿⣿⣿⣿⣿⢸⣿⣦⠙⣿⣿⣿⣿⣿⣿⣿
⡑⣑⣈⣻⢗⢟⢞⢝⣻⣿⣿⣿⣿⣿⣿⣿⠸⣿⠿⠃⣿⣿⣿⣿⣿⣿⡿
⡵⡈⢟⢕⢕⢕⢕⣵⣿⣿⣿⣿⣿⣿⣿⣿⣿⣶⣶⣿⣿⣿⣿⣿⠿⠋⣀
⣿⣿⣿⢻⣿⣿⣿⣿⠻⣿⣿⡿⠿⠿⠿⣿⠻⡿⠻⠿⣿⡟⠛⠛⢛⣿⣿
⣿⠏⣼⢸⡄⢿⢋⢸⣷⣇⢻⣿⣿⣿⣿⣿⡗⣥⠂⢦⣿⣭⢩⡍⣭⣽⣿
⣿⣾⣛⣸⣿⣾⣾⣈⣛⣠⣿⣤⣤⣤⣤⣼⣴⣫⣶⣌⣻⣋⣼⣇⣋⣼⣿

**Comment by GalaxyLHN** [comment]
粘合技术是非常好的网站程序设计，他的工作让我这一天开心，爱来自苏联

**Comment by wutizi** [comment]
太抽象了 真的太抽象了

**Comment by SpaceTimee** [comment]
关注粘连喵，举办粘连喵

**Reply by SpaceTimee** [reply]
吓到我了喵

**Reply by SpaceTimee** [reply]
你是一个，一个一个

**Reply by SpaceTimee** [reply]
热水q

**Comment by NenoSann** [comment]
你是一个一个一个

**Comment by FKcrown** [comment]
这是啥呀，举办粘连科技喵，举办粘连科技谢谢喵


|
|


|
|
***@***.***
|
On 4/2/2022 19:17，Space ***@***.***> wrote：

关注粘连喵，举办粘连喵

**Comment by Dreaminko** [comment]
早上好中國！
現在
我有冰淇淋
我很喜歡冰淇淋
但是
量子蟲洞
比冰淇淋
量子蟲
量子蟲洞
我最喜歡
所以...現在是音樂時間！
準備 1 2 3

**Comment by 14370o0** [comment]
LOVE IN IT.

**Comment by LinusLuo666** [comment]
非常喜欢

**Comment by 3170017873** [comment]
举办了喵

**Comment by minelcf** [comment]
已举办(

**Comment by NsCrosu** [comment]
狠↓狠→的举办

**Comment by simarenan** [comment]
绷

**Comment by slbcz** [comment]
我非常喜欢粘连的量子虫洞，每次使用量子虫洞都需要翻墙开GitHub，直到我发现这实在是太蠢了。有什么办法可以让我不必翻墙就可以使用粘连科技量子虫洞呢？那就是github app 于是经过了半天的翻墙，上谷歌，下应用，登录账号，加星标。终于，我可以不用翻墙就使用量子虫洞了，我找回了内心的平静。直到我发现这实在是太蠢了，每次使用量子虫洞都要打开GitHubApp，太麻烦了。有没有这样一套系统，可以让我在桌面直接打开粘连科技量子虫洞呢？很简单，我们只需要用chromium内核做一个简单的安卓app就好了，经过了半天的抄代码，抄代码，抄代码我发现这远远超出了我的能力范围。做一个简单的软件是很容易的，但 是对代码盲来说这是很难的。有什么东西是可以简单安装，简单打开的呢？那就是“将此网站作为应用安装”终于，我可以直接在桌面访问量子虫洞了，我找回了内心的平静。

**Comment by xiaochopin** [comment]
想你了😭
